draft-ietf-quic-recovery.md

```diff
@@ -102,7 +102,8 @@ In-flight:
 
 : Packets are considered in-flight when they have been sent
   and neither acknowledged nor declared lost, and they are not
-  ACK-only.
+  ACK-only.  Packets are still considered in-flight even if
+  their payloads no longer need to be delivered.
```

Comment:
```suggestion
  their payloads no longer need to be delivered in order to maintain the congestion window.
```

Reply:
I'm confused about what you're trying to add here?